dev.stewlab.PortaWar.json

```diff
@@ -0,0 +1,32 @@
+{
+    "app-id": "dev.stewlab.PortaWar",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "base": "org.godotengine.godot.BaseApp",
+    "base-version": "3.5",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "godot-runner",
```

Comment:
I don't see any such executable installed either.

Reply:
and how will it know what to run?

Reply:
Ah, I see what you mean.  Fixing now